lib/ember-cli/app.rb

```diff
@@ -95,10 +95,14 @@ def ember_path
 
     private
 
-    delegate :match_version?, :non_production?, to: Helpers
+    delegate :match_version?, to: Helpers
     delegate :configuration, to: EmberCLI
     delegate :tee_path, :npm_path, :bundler_path, to: :configuration
 
+    def non_production?
+      !Rails.env.production?
```

Comment:
Fair enough.

The problem I'm facing is that I've manually set `config.use_ember_middleware = true`, overriding the value of `non_production?`. This works for browsing the app, the EmberCLI app's `test/index.html` is no longer included in the build, since `non_production?? "development" : "production"` is evaluating to `"production"`.

This is breaking the work done in `0.1.7`